README.md

```diff
@@ -56,6 +56,7 @@ These are some of patterns definitions to help us to keep a default configuratio
 Name | Description
 --- | ---
 [Demo](https://juggernaut-demo.web.app/) | *Examples of how to use Juggernaut*
+[Demo Repository](https://github.com/SoftboxLab/juggernaut-demo) | *Juggernaut Demo source code*
```

Comment:
Ao invés de `Juggernaut Demo source code` não ficaria melhor `Source code of Juggernaut demo`? Ficaria mais correto assim no inglês @vilaboim

Reply:
Acredito que os dois são corretos.